searchcore/src/vespa/searchcore/proton/docsummary/docsumcontext.cpp

```diff
@@ -137,14 +137,16 @@ DocsumContext::createSlimeReply()
 DocsumContext::DocsumContext(const DocsumRequest & request, IDocsumWriter & docsumWriter,
                              IDocsumStore & docsumStore, const Matcher::SP & matcher,
                              ISearchContext & searchCtx, IAttributeContext & attrCtx,
-                             search::IAttributeManager & attrMgr, SessionManager & sessionMgr) :
+                             IAttributeManager & attrMgr, const IAttributeExecutor & attrExec,
```

Comment:
Why separate attrExec here ? IAttributeManager inherits from IAttributeExecutor, so attrExec can be derived from attrMgr.

Reply:
Tried, but it is search::IAttributeManager.